server/app/views/components/QuestionBank.java

```diff
@@ -108,17 +108,31 @@ private FormTag questionBankPanel() {
     SvgTag filterIcon = Icons.svg(Icons.SEARCH).withClasses(Styles.H_4, Styles.W_4);
     DivTag filterIconDiv =
         div().withClasses(Styles.ABSOLUTE, Styles.ML_4, Styles.MT_3, Styles.MR_4).with(filterIcon);
-    DivTag filterDiv = div().withClasses(Styles.RELATIVE).with(filterIconDiv).with(filterInput);
+    DivTag filterDiv =
+        div().withClasses(Styles.MB_2, Styles.RELATIVE).with(filterIconDiv, filterInput);
     contentDiv.with(filterDiv);
+    contentDiv.with(
+        div()
+            .with(
+                div()
+                    .with(
+                        p("Not finding a question you're looking for in this list?"),
+                        div()
+                            .withClass(Styles.FLEX)
+                            .with(
+                                div().withClass(Styles.FLEX_GROW),
+                                CreateQuestionButton.renderCreateQuestionButton()))));
```

Comment:
Screenshot shows that button rendered underneath "Not finding a question" text while mock has screenshot to be on the right and text on the left.

Reply:
Indeed. This was something @rahaghas and I live-built and were ok with the position being slightly different from the mocks. I called this out in the description.